README.md

```diff
@@ -0,0 +1,34 @@
+# BETTER BUILDS
+This React website is for users to submit and review PC builds that will be posted to PostgreSQL database. Users can also edit their reviews and vote on the reviews of other users with Express backend, running on a NodeJS environment.
+## Contributors: 
+Nick DeRis - github.com/dericholas
+Nick Kwiatkowski - github.com/nickkwiat
+Robert Le - github.com/fuzzyphantasm
+Corey Pierce - github.com/cornfrog
+Marlon Sanchez - github.com/MarlonS419 
+## Installation
+```
+git clone https://github.com/MarlonS419/better-builds
```

Comment:
we suggest setting the type for code blocks made with triple backticks - since this is a command expected to be run from the terminal we can add the word `shell` after the first opening triple back ticks. (the example below uses quotation marks instead of backticks to avoid formatting of the code block)
```
'''shell
```
^^ still use backticks, just add `shell`